src/gui/EditWidgetIcons.cpp

```diff
@@ -528,3 +553,31 @@ void EditWidgetIcons::updateRadioButtonCustomIcons()
 {
     m_ui->customIconsRadio->setChecked(true);
 }
+
+void EditWidgetIcons::confirmApplyIconTo(QAction* action)
+{
+    ApplyIconToOptions selectedOption = action->data().value<ApplyIconToOptions>();
+    QAction* actualAction = action;
+
+    // If no other icons are to be overwritten, no popup is shown
+    if (selectedOption != ApplyIconToOptions::THIS_ONLY) {
+        auto result = MessageBox::question(this,
```

Comment:
I find this question to be superfluous now that the checkbox was turned into a combo button. It is quite clear what is going on when you use the combo box. Recommend removal.